scss/layout/_grid.scss

```diff
@@ -68,6 +69,16 @@
   }
 }
 
+.slab--primary .grid {
+  a {
```

Comment:
Can you point me to the other pages that use `.grid` inside a `.slab` that you're seeing at issue, @noahmanger ? It could be easy and logical to eliminate any special instances, but these opportunities are only things I learn about through implementation like this. Glad to give it a go if we can surface the instances it would interfere with.